requirements.txt

```diff
@@ -8,9 +8,12 @@ click==8.1.6
 contourpy==1.1.0
 cycler==0.11.0
 docopt==0.6.2
+dvc==3.15.3
```

Comment:
I see you added DVC as a package in the project. When I installed DVC as a python package on my machine, I had some problems, which were solved when installing it through apt.

Reply:
Maybe it's a good idea to not have it as package.

Reply:
done